Assets/_Scripts/TextAreaManager.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class TextAreaManager : MonoBehaviour
+{
+    [SerializeField] bool preWrite;
+    [SerializeField] bool midWrite;
+    [SerializeField] bool postWrite;
+
+    [SerializeField] TMP_Text dialogue;
+    //[SerializeField] DialogueSO Dialogue;
+
+    void Update()
+    {
+        if (preWrite)
```

Comment:
To make handling the "next" button easier. With these states, the next button can dynamically change to a skip button, to have the text written immediately instead of it being animated, once the text fully displayed, the button can dynamically change back to a next button. I think of it as the "polite" way to put code something for the user, at the end of the day not every user would want to keep looking at the same animation over and over again.

Reply:
Mmm, are we going to have it be a next button, or a lack of a button press? 
What I mean is, is there a need for an actual button when going to the next screen? Or can we just make it so they can click anywhere on the screen, and then we have a static "back" button somewhere on the screen for them to use? 
I guess this is a UX question for Susan.

Reply:
Even if the user gets to click anywhere there still has to be a button, only its invisible.

Reply:
So these are states that the script would cycle through. Then the button would behave differently according to the state. There is a million different ways to achieve this but this is the easiest in my opinion.

Reply:
K. I am good with this. Are you good with keeping ownership of this system throughout?

Reply:
Just that since you have designed it, it is your responsibility to get it to work? Is that alright?

Reply:
We are a team, if something from your end breaks its my duty as your teammate to try my best to fix it, same goes the other way, if you think that that system is overkill or over-engineered then I can simply do it another way that we both agree on. After all there is a million different ways to get something like that going.

Reply:
But also at the same time, we need to avoid stepping on each others toes with all of this. If there is something that is breaking everything, I will try to fix it, but I also know from time on several software teams that my fix may cause more problems if I don't perfectly understand your implementation. So if it doesn't break everything, and we have time, I will defer to you to fix it.